react-native-gutenberg-bridge/ios/Gutenberg.swift

```diff
@@ -63,6 +63,14 @@ public class Gutenberg: NSObject {
         if let translations = dataSource.gutenbergTranslations() {
             initialProps["translations"] = translations
         }
+
+        if let slug = dataSource.siteSlug {
+            initialProps["siteSlug"] = slug
+        }
+
+        if let extraHeaders = dataSource.extraHTTPHeaders {
+            initialProps["extraHTTPHeaders"] = extraHeaders
```

Comment:
I would try to keep auth tokens away from initial props since react native seems to aggressively log those, and we shouldn’t log tokens

Reply:
Yes that makes sense. This was the easier way to make it work.

And also we will need to refresh it when the user switches the blog from the editor, avoiding fetching it on every network call.